src/main/java/com/connexta/multiintstore/controllers/StorageController.java

```diff
@@ -0,0 +1,10 @@
+package com.connexta.multiintstore.controllers;
+
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class StorageController {
```

Comment:
❓Is this the class that will be handling callbacks from the Transformation Service? Not sure I understanding the different between `StorageController` and `StorageManager`. Not a blocker for this PR. Just want to understand what the intention is.

Reply:
Still an amateur at Spring, so maybe the Manager class isn't needed. The controller is where we would handle all of the REST calls. I can remove the Manager since it doesn't make much sense right now.

Reply:
No, it's fine. Thanks for the explanation. It will probably make more sense later.